src/Microsoft.DotNet.Build.CloudTestTasks/FilterForUpload.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.DotNet.Build.CloudTestTasks
+{
+    public sealed class FilterForUpload : Task
+    {
+        [Required]
+        public string TestOutputDirListsLocation { get; set; }
```

Comment:
There needs to be a comment explaining what this file is and what format it's supposed to be in, since it appears to be a text file of some sort with very strict requirements on it's format, or this task will behave very strangely.